packages/kTThreads/macros/old/kThreadTest.C

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Do you want to keep this "old" macro/directory?  If so, can you rather rename it a proper name?

Reply:
I will delete it, the removal commit was lost when I moved from a clone to fork directory.